components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -54,6 +54,8 @@ export class CreateObjectModalComponent implements OnInit, OnChanges {
 
   onProjectChecked(project: ProjectChecked): void {
     this.projects[project.id].checked = project.checked;
+    const projectsSelected = Object.values(this.projects).filter(p => p.checked);
+    this.createForm.controls.projects.setValue(projectsSelected.map(p => p.id));
```

Comment:
This is a bit indirect.... assuming that the _keys_ are the project IDs, we could do
```typescript
const projectsSelected = Object.keys(this.projects).filter(id => this.projects[id].checked);
this.createForm.controls.projects.setValue(projectsSelected);
```
or am I missing something?